bindings/go/evmc/host.go

```diff
@@ -76,6 +76,7 @@ type TxContext struct {
 	GasLimit   int64
 	Difficulty Hash
 	ChainID    Hash
+	BaseFee    Hash // EIP-1559, EIP-3198
```

Comment:
Do we need to keep these comments here? If so, perhaps for the older ones (such as ChainID) too?